app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_name, base_filename, new_filename = command.split()
+    if base_filename == new_filename:
+        return
+    if command_name == "cp":
+        with open(base_filename, "r") as source:
+            content = source.read()
+
+        with open(new_filename, "w") as destination:
+            destination.write(content)
```

Comment:
do it according to the format 


![image](https://user-images.githubusercontent.com/104218550/220900209-6346e7b0-9ab2-4f33-983f-8830c7e22047.png)